src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  &__container {
+    width: 80vmin;
+    height: 80vmin;
+    border: dotted 1vmin #000;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use logical properties for borders, like `border-style`, `border-width`, and `border-color` separately instead of shorthand for better readability.